browser/src/control/Control.StatusBar.js

```diff
@@ -464,11 +464,12 @@ class StatusBar extends JSDialog.Toolbar {
 			this.updateHtmlItem('RowColSelCount', state ? state : _('Select multiple cells'), !state);
 		}
 		else if (commandName === '.uno:InsertMode') {
-			this.updateHtmlItem('InsertMode', state ? L.Styles.insertMode[state].toLocaleString() : _('Insert mode: inactive'), !state);
+			this.updateHtmlItem('InsertMode', state ? L.Styles.insertMode[state].toLocaleString() : ' ', !state);
 
 			$('#InsertMode').removeClass();
 			$('#InsertMode').addClass('jsdialog ui-badge insert-mode-' + state);
-			$('#insertmode-container').attr('default-state', state || null);
+			var isDefaultState = state === 'true' || state === '';
+			$('#insertmode-container').attr('default-state', isDefaultState || null);
```

Comment:
just a nitpick for the future: could we do not use jQuery in the new code, we want to remove it :)
```
const container = document.getElementById('insertmode-container');
container.setAttribute(...)
```

this file has it anyway so we can merge like that